### PR TITLE
Added to readme.txt

### PR DESCRIPTION
**Pull request by adammoy2**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,8 +1,22 @@
+=====BASIC SET-UP =====
+* Create .env file; the contents are described in ENVIRONMENT VARIABLES
+* Create a certificate and key. Place those in cert/cert/ (by default)
```
Comment (pranaygp) [inline readme.txt]: A link to instructions on how to create a certificate and key could be useful

File: readme.txt [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
+    You will need to provide this information to the IdP.
```
Comment (pranaygp) [inline readme.txt]: What's IdP?

File: readme.txt (hunk 1 above)
Comment (adammoy2) [inline readme.txt]: I provided the console commands in the next section. I could still provide a link if you want.

Comment (adammoy2) [inline readme.txt]: Ok, I think I added some links... I think.

**Pull request merged**